### ISSUE TITLE
[BUG] Bump to Angular 15

### ISSUE THREAD

**Issue by maxcodefaster** [TinkoffCreditSystems/micro-sentry#40]

**Comment by bufke** [comment]
With Angular 16 out soon, what do you think of setting the peerDependecy to ">14.0.0"

The Angular part of this project is very small. IMO there are better uses of time than updating peerDependency twice a year for something that is very rare to break. I'm happy to update the tooling to Angular 15 and just setting it to >15 if desired.

**Comment by maxcodefaster** [comment]
> 

Yeah just set it so that we do not have to do "npm i -f" cause of dependency conflicts. Project should work just as fine. It is just the config.

**Comment by Anectem** [comment]
Any news?
